src/init.cpp

```diff
@@ -418,7 +418,7 @@ void SetupServerArgs(NodeContext &node) {
         "-min", "-resetguisettings", "-rootcertificates=<file>", "-splash",
         "-uiplatform",
         // TODO remove after the November 2020 upgrade
-        "-axionactivationtime"};
+        "-axionactivationtime", "-exodusactivationtime"};
```

Comment:
This other flag isn't being used except by the test suite right?

Reply:
Yes, seems like it.

Reply:
Rather than adding this flag, it might be good to find & replace all axionactivation times. There's stuff in a variety of files that cause the test on (for example, this diff) to run tests with the activated changes.